### PR TITLE
Added camType Prop

### PR DESCRIPTION
**Pull request by himelbrand**
the type prop of Camera was missing, my team and I are using this package, but we need the scanner to use the front camera.
can be 'front' or 'back' default is 'back'

if approved please update the npm package as well

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export default class QRCodeScanner extends Component {
     reactivateTimeout: PropTypes.number,
     fadeIn: PropTypes.bool,
     showMarker: PropTypes.bool,
+    camType: PropTypes.string,
```
Comment (moaazsidat) [inline index.js]: can we make this more explicit using `PropTypes.oneOf(['back', 'front'])`

Comment (moaazsidat) [reply index.js]: also can we rename this to `cameraType`, clearer if variable names aren't shortened.

File: index.js [hunk 2]
```diff
@@ -46,6 +47,7 @@ export default class QRCodeScanner extends Component {
     reactivateTimeout: 0,
     fadeIn: true,
     showMarker: false,
+    camType: 'back',
```
Comment (moaazsidat) [inline index.js]: same as above for name change to `cameraType`

**Comment by moaazsidat** [comment]
Can you also add documentation for the `cameraType` props in the README, thanks!

**Comment by himelbrand** [comment]
I committed the changes you requested, hope it's fine

**Comment by moaazsidat** [comment]
released in [v0.0.23](https://www.npmjs.com/package/react-native-qrcode-scanner), thanks for adding this!

**Pull request merged**